### PR TITLE
Add changelog for v10.2.0-rc.1

### PR DESCRIPTION
**Pull request by ank27**
### Summary of changes
- Add changelog for v10.2.0-rc.1

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
    
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
 - [ ] If this PR is a `v10.[version]` release branch fix / enhancement, merge it to `main` firstly and then port to `v10.[version]` release branch.

Fixes: < Link to related issues that will be fixed by this pull request, if they exist >

PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

### CODE REVIEW

**Comment by kiryldz** [comment]
@ank27 please change base branch to `main` first thing and then we'll port it to v10.2 branch.

**Comment by kiryldz** [comment]
> @ank27 please change base branch to `main` first thing and then we'll port it to v10.2 branch.

However not sure, perhaps it's indeed more logical to do it that way...

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+
+## Features ✨ and improvements 🏁
+* Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
+
+## Bug fixes 🐞
+* Do not allow to use one associatedFeatureId with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
+* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
+* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927))
+* Reduce drag sensitivity around and above the horizon. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
+* Fix Fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
```
Comment (kiryldz) [inline CHANGELOG.md:15]:
```suggestion
* Fix fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
+
+## Bug fixes 🐞
+* Do not allow to use one associatedFeatureId with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
+* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
+* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927))
+* Reduce drag sensitivity around and above the horizon. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
```
Comment (kiryldz) [inline CHANGELOG.md:14]: I guess it's not needed as we have #927 
@pengdev could you please advise here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```
Comment (kiryldz) [inline CHANGELOG.md:8]: And move to bugs as cancelable QRF were already introduced before
```suggestion
* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

**Comment by ank27** [comment]
> > @ank27 please change base branch to `main` first thing and then we'll port it to v10.2 branch.
> 
> However not sure, perhaps it's indeed more logical to do it that way...

Yeah, only for the release process, we could first merge v10.2 then port changelog to main. I think this is more logical, wdyt?

File: CHANGELOG.md (hunk 2 above)
Comment (pengdev) [reply CHANGELOG.md:14]: yes, it's the gl-native part fix of #927, maybe we can merge this entry with #927

Comment (pengdev) [reply CHANGELOG.md:14]: something like 

* Fix gestures at high camera pitch near horizon line. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925), [#927](https://github.com/mapbox/mapbox-maps-android/pull/927))

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```
Comment (kiryldz) [inline CHANGELOG.md:8]: move to bugs, it's not a feature
`Features ✨ and improvements 🏁` section is not needed

File: CHANGELOG.md (hunk 3 above)
Comment (yunikkk) [reply CHANGELOG.md]: Is it more a bugfix, nor an improvement / feature?

Comment (yunikkk) [reply CHANGELOG.md]: @kiryldz probably not a `suggestion message` but `replaceWith` template or similar name

**Review by pengdev — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,18 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Bug fixes 🐞
+* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```
Comment (yunikkk) [inline CHANGELOG.md:8]:
```suggestion
* Fix `replaceWith` template to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

**Review by yunikkk — APPROVED** [review]
Small suggestion

**Review by knov — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,19 @@
...
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Bug fixes 🐞
+* Fix `replaceWith` template to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
+* Do not allow to use one `associatedFeatureId` with multiple view annotations. ([#896](https://github.com/mapbox/mapbox-maps-android/pull/896))
+* Fix an issue where shove gesture was not detected when angle between touch points are not horizontal. ([#875](https://github.com/mapbox/mapbox-maps-android/pull/875))
+* Fix gestures at high camera pitch near horizon line. ([#927](https://github.com/mapbox/mapbox-maps-android/pull/927), [#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
+* Fix fading tiles layout visibility issue on globe view projection zooming in/out. ([#925](https://github.com/mapbox/mapbox-maps-android/pull/925))
+* Add config to enable simultaneous zoom and rotate gesture. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))
```
Comment (pengdev) [inline CHANGELOG.md:13]: shall we mention the default behaviour is changed(fixed) as we now by default allow simultaneous zoom and rotate gesture?

Comment (pengdev) [reply CHANGELOG.md:13]:
```suggestion
* Allow simultaneous zoom and rotate gesture by default and add `GesturesSettings#simultaneousRotateAndPinchToZoomEnabled` config option. ([#885](https://github.com/mapbox/mapbox-maps-android/pull/885))
```

Comment (ank27) [reply CHANGELOG.md:13]: should we mention it in changelog or in release note?

Comment (pengdev) [reply CHANGELOG.md]: I think we should

**Pull request merged**